authors/apps/utils/app_util.py

```diff
@@ -0,0 +1,31 @@
+from django.test import TestCase, RequestFactory
+from authors.apps.authentication.views import (
+    RegistrationAPIView, VerificationAPIView, LoginAPIView)
+import json
+from minimock import Mock
+import smtplib
+
+
+class UtilClass():
+    def __init__(self):
+        self.factory = RequestFactory()
+        smtplib.SMTP = Mock('smtplib.SMTP')
+        smtplib.SMTP.mock_returns = Mock('smtp_connection')
+
+    def get_reg_data(self, user):
+        request = self.factory.post(
+            '/api/users/', data=json.dumps(user), content_type='application/json')
+        response = RegistrationAPIView.as_view()(request)
```

Comment:
My only issues remain with using this method here, but other than that, it should all be good.

Reply:
Let's work with this for now.